.refitter

```diff
@@ -0,0 +1,14 @@
+{
+    "openApiPath": "docs/schema/V1/swagger.verified.json",
```

Comment:
Blir refitter bare kalt kun en gang? Eller gjør vi det hver gang API endrer seg? Isåfall burde vi kanskje ha det som en del av bygg-steget? 🤔

Reply:
akkurat nå blir den generert av en test lignene swaggerSnapshot testen. men holder på å prøver meg litt frem for å finne en løsning